core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -20,8 +20,9 @@ trait RTS {
    */
   final def unsafeRun[E, A](io: IO[E, A]): A = unsafeRunSync(io) match {
     case ExitResult.Completed(v)       => v
-    case ExitResult.Terminated(Nil)    => throw Errors.TerminatedFiber
-    case ExitResult.Terminated(t :: _) => throw t
+    case ExitResult.Terminated(Nil, Nil)    => throw Errors.TerminatedFiber
+    case ExitResult.Terminated(Nil, t :: _) => throw t
```

Comment:
Don't think it makes sense to throw this here instead of `TerminatedFiber` since it's not the cause of termination.